examples/common/lib/utils.sh

```diff
@@ -139,6 +139,39 @@ function wait_for_healthy_shard() {
 	wait_for_shard_vreplication_engine "${keyspace}" "${shard}"
 }
 
+# Stop the specified binary name using the provided PID file.
+# Example:
+#  stop_process "vtadmin-web" "$VTDATAROOT/tmp/vtadmin-web.pid"
+function stop_process() {
+	if [[ -z ${1} || -z ${2} ]]; then
+		fail "A binary name and PID file must be specified when attempting to shutdown a process"
+	fi
+
+	local binary_name="${1}"
+	local pidfile="${2}"
+	local pid=""
+	local wait_secs=90
+
+	if [[ -e "${pidfile}" ]]; then
+		pid=$(cat "${pidfile}")
+		echo "Stopping ${binary_name}..."
+		kill "${pid}"
+		# Wait for the process to terminate
+		for _ in $(seq 1 ${wait_secs}); do
+			if ! ps -p "${pid}" > /dev/null; then
+      				break
+			fi
+			sleep 1
+		done
+	else
+		echo "Skipping stopping ${binary_name} because the specified PID file (${pidfile}) does not exist."
+	fi
+
+        if ps -p "${pid}" > /dev/null; then
+                fail "Timed out after ${wait_secs} seconds waiting for the ${binary_name} using PID file ${pidfile} to terminate"
+        fi
```

Comment:
There is a mix of spaces and tabs on line `162`, `170`, `171` and `172`, let's use tabs only to be consistent

Reply:
I moved this and noticed/corrected it then.